policy/modules/system/systemd.te

```diff
@@ -1417,3 +1417,516 @@ optional_policy(`
+# AVC avc:  denied  { rename } for  pid=1000 comm="systemd-homewor" name=".#.identity9bbf460fb708fc96" dev="dm-15" ino=137 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:object_r:user_home_dir_t:s0 tclass=file permissive=1
+# AVC avc:  denied  { setattr } for  pid=1000 comm="systemd-homewor" name=".#.identity9bbf460fb708fc96" dev="dm-15" ino=137 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:object_r:user_home_dir_t:s0 tclass=file permissive=1
+# AVC avc:  denied  { unlink } for  pid=1000 comm="systemd-homewor" name=".identity" dev="dm-15" ino=136 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=unconfined_u:object_r:user_home_t:s0 tclass=file permissive=1
+# AVC avc:  denied  { write open } for  pid=1000 comm="systemd-homewor" path="/home/testuser/.#.identity9bbf460fb708fc96" dev="dm-15" ino=137 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=system_u:object_r:user_home_dir_t:s0 tclass=file permissive=1
+manage_files_pattern(systemd_homed_t, user_home_dir_t, user_home_dir_t)
+
+# AVC avc:  denied  { read } for  pid=1000 comm="systemd-homewor" name=".identity" dev="dm-15" ino=136 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=unconfined_u:object_r:user_home_t:s0 tclass=file permissive=1
+# AVC avc:  denied  { open } for  pid=1000 comm="systemd-homewor" path="/run/systemd/user-home-mount/testuser/.identity" dev="dm-15" ino=136 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=unconfined_u:object_r:user_home_t:s0 tclass=file permissive=1
+# AVC avc:  denied  { unlink } for  pid=1000 comm="systemd-homewor" name=".identity" dev="dm-15" ino=136 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=unconfined_u:object_r:user_home_t:s0 tclass=file permissive=1
+userdom_manage_user_home_content_files(systemd_homed_t)
+
+# AVC avc:  denied  { getattr } for  pid=1000 comm="systemd-homewor" path="/home/testuser.homedir/.mozilla" dev="dm-14" ino=171140546 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=unconfined_u:object_r:mozilla_home_t:s0 tclass=dir permissive=0
+# AVC avc:  denied  { rmdir } for  pid=1000 comm="systemd-homewor" name=".mozilla" dev="dm-14" ino=171140546 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=unconfined_u:object_r:mozilla_home_t:s0 tclass=dir permissive=0
+# AVC avc:  denied  { write } for  pid=1000 comm="systemd-homewor" name=".mozilla" dev="dm-14" ino=171140546 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=unconfined_u:object_r:mozilla_home_t:s0 tclass=dir permissive=0
+# AVC avc:  denied  { remove_name } for  pid=1000 comm="systemd-homewor" name="plugins" dev="dm-14" ino=18877888 scontext=system_u:system_r:systemd_homed_t:s0 tcontext=unconfined_u:object_r:mozilla_home_t:s0 tclass=dir permissive=0
+optional_policy(`
+	gen_require(`
+		type mozilla_home_t;
+	')
+	mozilla_read_user_home_files(systemd_homed_t)
+	allow systemd_homed_t mozilla_home_t:dir { remove_name rmdir write };
+')
+
```

Comment:
Not sure what is going on here and what business homework has with ~/.mozilla. Also probably better to not reference mozilla_home_t directly here.